packages/rancher-istio/charts/values.yaml

```diff
@@ -5,7 +5,7 @@ forceInstall: false
 
 installer:
   repository: rancher/istio-installer
-  tag: 1.7.3-rancher1
+  tag: 1.7.3-rancher2
 
```

Comment:
I thought we added a grafana flag for on or off, but I guess not. It just runs if its able. We might want to add that later.

Reply:
Yeah, it seemed like grafana won't install unless rancher-monitoring is installed and you would set grafana enabled there....